isfolkid.sql

```diff
@@ -0,0 +1,34 @@
+-- Hversu margar aðalpersónur eru í bókabálkinum?
+SELECT SUM(LENGTH(characters) - LENGTH(REPLACE(characters, ',', '')) + 1) AS total_main_characters
+FROM books
+WHERE characters IS NOT NULL;
+
+--Hversu margar persónur eru í hverri bók
+SELECT
+    is_title,
+    LENGTH(characters) - LENGTH(REPLACE(characters, ',', '')) + 1
+FROM books;
+
+-- Hversu oft kemur Þengill fyrir í bókunum
+SELECT COUNT(*) AS total_occurrences
+FROM books
+WHERE characters LIKE '%Þengill%';
+
+-- Hversu margir af Paladín ættinni
+SELECT COUNT(*) FROM family WHERE name LIKE '%Paladín%';
+
+-- Hversu algengur er illi arfurinn af þeim sem eru útvalin
+SELECT COUNT(*) FROM family WHERE chosen_one='evil';
+
+-- Hver er fæðingatíðni kvenna
+SELECT birth, COUNT(*) AS total_females
+FROM family
+WHERE gender = 'F' AND birth IS NOT NULL
+GROUP BY birth
+ORDER BY birth ASC;
```

Comment:
Þetta er að telja fjölda stúlkubarna sem fæðast á hverju ári sem barn fæðist. En ekki hver er meðal fæðingartíðnin hjá konunum í fjölskyldunni. 

```suggestion 
SELECT AVG(child_count) AS mean_birth_rate
FROM (SELECT COUNT(child) AS child_count
      FROM children
      WHERE parent IN (SELECT id FROM family WHERE gender = 'F')
      GROUP BY parent);
```